srcpkgs/boot-66serv/files/runit

```diff
@@ -0,0 +1,31 @@
+[main]
+@type = classic
+@version = @VERSION@
+@description = "runit"
+@user = (root)
+
+[start]
+@build = custom
+@shebang = ( "/bin/sh" )
+@execute = (
+PATH=/usr/bin:/usr/sbin
```

Comment:
Why not only `/usr/bin`? And maybe `/usr/local/bin`.

Reply:
I follow the /etc/runit/2 script. It only enables these these two - one, as /sbin is a symlink to /usr/bin.

Reply:
Left only /usr/bin. Thank you!